tests/chainer_tests/distributions_tests/test_kldivergence.py

```diff
@@ -63,6 +70,18 @@ def make_normal_dist(self, is_gpu=False, use_log_scale=False):
             params = self.encode_params({"loc": loc, "scale": scale}, is_gpu)
         return distributions.Normal(**params)
 
+    @testing.with_requires('numpy>=1.11')
```

Comment:
Could you explain why new numpy is required?

Reply:
In old numpy, this error occurs,
```
>>> import numpy
>>> p = numpy.random.normal(size=(3)).astype(numpy.float32)
>>> p = numpy.exp(p)
>>> p /= p.sum()
>>> numpy.random.choice(3, p=p)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "mtrand.pyx", line 1083, in mtrand.RandomState.choice (numpy/random/mtrand/mtrand.c:10106)
ValueError: probabilities do not sum to 1
```